doc/nrf/working_with_nrf/nrf70/developing/powersave.rst

```diff
@@ -143,13 +143,57 @@ The following figure illustrates the unicast frame data retrieval mechanism in W
    WMM Power Save mode
 
 Legacy Power Save mode is the default option in nRF70 Series devices.
-The user or application can configure the WMM mode using the ``NET_REQUEST_WIFI_PS_MODE`` network management API.
+The user or application can configure the WMM mode using the ``NET_REQUEST_WIFI_PS`` network management API.
 The WMM mode does not deliver a significant performance or power difference when compared to Legacy mode in Dynamic Power Save operation.
 
 The average power consumption of the device is affected by the DTIM period.
 The typical value is three beacons, for example, 307 ms for a beacon period of 100-time units.
 The higher DTIM period results in increased power saving and higher latency to the application data.
 
+.. _ug_nrf70_developing_powersave_extended_ps:
+
+Extended Power Save
+*******************
+
+The Listen Interval based Power Save mode is supported to enable devices to sleep longer than DTIM period in absence of TWT session.
+The device wake up interval is adjusted to nearest multiple of DTIM Period.
+For example, listen interval of 10, device wakes up for every 9th beacon if DTIM is 3 and 8th beacon if DTIM is 4.
+Device wakes up for configured Listen interval instead of DTIM beacon and will miss chance of receiving broadcast and multicast frames which get scheduled after DTIM
+beacon.
+Listen interval-based power needs to be enabled only if application can handle loss broadcast frames.
+Chances of losing broadcast frames increases with higher listen interval.
+Following parameters control functionality of extended power save operation.
```

Comment:
```suggestion
The following parameters control the functionality of the extended Power Save operation.
```

Reply:
I think "power save" operations instead of capitalizing **Power Save** as we are talking about operations not mode here?